test/mobile/op_deps/main.cc

```diff
@@ -13,7 +13,7 @@ int main() {
   at::call_DD_op(input);
   at::call_EE_op(input);
   at::call_FF_op(input);
-  callOp("quantized::t_add", "", input, input, 1.0, 0);
-  callOp("quantized::t_add_relu", "", input, input, 1.0, 0);
+  callOp_slow("quantized::t_add", "", input, input, 1.0, 0);
+  callOp_slow("quantized::t_add_relu", "", input, input, 1.0, 0);
```

Comment:
These calls don't matter, we're just trying to demonstrate that manually called ops get tracked. You can also just switch this to whatever the correct api is.

Reply:
I could probably switch all of those to the correct API instead, which would be the better approach, but I only had time for this half-way solution. I wanted to at least make sure people don't continue to copy&paste this to new locations, it started with one implementation and now we have 3 `callOp` implementations.